assessment_helpers/to_csv.py

```diff
@@ -0,0 +1,41 @@
+#This script formats the "GAdata.txt" file (created by trainGeneticAlgorithm.py) as a .csv file named "clean_ga_data.csv"
+#Note: make sure "GAdata.txt" is named exactly that and is in the same directory as this script
+#Note: if "clean_ga_data.csv" already exists in this directory it will be overwritten
+
+import csv
+
+ga_data = open("GAdata.txt", "r")
+
+clean_ga_data = open("clean_ga_data.csv", "w")
```

Comment:
I think we should try to avoid hard coding filenames into these kinds of scripts as they make the scripts pretty limited. See my comment on plot_csv.py for one way to add command line arguments.

Reply:
Word, will pattern match the argument parser stuff you put in the plotting script and add it in here